src/com/company/Main.java

```diff
@@ -6,9 +6,55 @@
 
 public class Main {
 
-    public static void main(String[] args) {
+    public static void createBank(final Scanner scanner, final List<Bank> banks) {
+        System.out.println("Create new bank");
+        final String nameBank = scanner.nextLine();
+        System.out.println("Enter the address");
+        System.out.println("City:");
```

Comment:
Why not creating a separated method for Address creation only?

Reply:
Add :)